styles.css

```diff
@@ -2,6 +2,8 @@ body {
     background-color: #f9fafc;
 
     font-family: 'Open Sans', sans-serif /*"Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif*/;
+/*     sans-serif is a property for the Open Sans font. Sans serif means that the text will not have any free edges hanging off, compare this with how
+    Times-New Roman looks where it has little edges coming off each end of the line. You can see this by googling serif vs sans-serif fonts */
         font-weight: normal;

```

Comment:
also, these font-weight and style of normal arent doing anything because the default for these is normal.